serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,156 @@
-//= Building Workflow Images with Tekton Pipelines
\ No newline at end of file
+= Building Workflows Images with Tekton
+:compat-mode!:
+// Metadata:
+:description: Building Workflow Images with Tekton
+:keywords: kogito, workflow, serverless, kubernetes, minikube, openshift, containers, tekton, pipeline
+
+This document describes how to build a Workflow Application on a Kubernetes cluster using the Tekton pipeline.
+
+
+.Prerequisites
+You have a git repo containing your workflow and the dockerfile to build the workflow as a docker/podman image
+and a namespace where to install Tekton i.e. `custom-namespace`.
+
+[NOTE]
+====
+
+If you are using Openshift, you can install the Red Hat Pipeline Operator. If you are using Minikube, you can follow this link:https://github.com/tektoncd/pipeline/blob/main/docs/developers/local-setup.md#using-minikube[tekton guide for Minikube].
+If you are using Kind you can follow this link:https://github.com/tektoncd/plumbing/tree/main/hack[Tekton guide for Kind]. If you want to use it on a plain kubernetes cluster, you can follow this link:https://tekton.dev/docs/pipelines/install/[Tekton guide for kubernetes].
+====
+
+
+== Create the pipeline
+In the `custom-namespace` namespace, in the pipelines menu, open the pipeline item, create it, and copy the following yaml file.
+It contains a pipeline to clone your image workflow repository. You can build the image and push it to the local image registry.
```

Comment:
will it clone a container image or the git repository to build the operator?

Reply:
It clone the git repo and then build the image with the dockerfile found in the repo, not the repo of the operator, this is for general images, but works in the same way if you use the operator git repo as a parameter

Reply:
ok